pyproject.toml

```diff
@@ -17,6 +17,7 @@ license = "Apache-2.0"
 packages = [{ include = "lm_buddy", from = "src" }]
 
 [tool.poetry.dependencies]
+# Core
```

Comment:
since this is a fairly large API change, bump the version of the package?

Reply:
It's going into a dev branch, so a version bump will come at the end when merging into `main`.

Reply:
I've bumped it now actually, Ill leave the package as a 0.2.0 for the subsequent PRs